src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(instance, num) {
+      instance.result += num;
+    },
+    subtract(instance, num) {
```

Comment:
```suggestion
    result: 0,
    
    add(instance, num) {
      instance.result += num;
    },
    
    subtract(instance, num) {
```
Don't forget to add newlines for better readability